.github/workflows/test-and-check.yml

```diff
@@ -109,7 +109,7 @@ jobs:
         run: node packages/wrangler/src/__tests__/test-old-node-version.js error
 
   test-vp:
-    if: github.head_ref == 'changeset-release/main' || contains(github.event.*.labels.*.name, 'every-os' )
+    if: github.head_ref == 'changeset-release/main' || contains(github.event.*.labels.*.name, 'every-os' || contains(github.event.*.labels.*.name, 'every-os-unit' )
```

Comment:
Do you still want the `every-os-unit` label? Just checking, happy to keep  it :)